app/resources/logout.html.template

```diff
@@ -0,0 +1,10 @@
+{% extends "app-base.html.template" %}
+{% load i18n %}
+{% block content %}
+<div>
+  {% if user %}
```

Comment:
This template is only used when the user object exists, right? Then you don't need this "if user".